common/oauthTokenManager.go

```diff
@@ -42,6 +42,9 @@ import (
 	"github.com/Azure/go-autorest/autorest/date"
 
 	"github.com/Azure/go-autorest/autorest/adal"
```

Comment:
We still have reference to adal.Token in the code which shall be removed as adal package itself is deprecated now and as part of our Track-2 migration we have moved to msal. Refresh() method in this file still relies on adal.Token so we might need wider changes to remove adal dependency completely here.

Reply:
Unfortunately we need to keep this to maintain compatibility with storage explorer. We can take up an item to work with them to change the format of the token they send but that may take another release cycle or 2. 

For now to remove the dep I should be able to manually copy the adal.Token type into our codebase, so I will do that.